src/integrationTest/java/uk/gov/hmcts/reform/sendletter/helper/FtpHelper.java

```diff
@@ -30,7 +30,8 @@ private static FtpClient getClient(int port, boolean verified) throws IOExceptio
             client.addHostKeyVerifier((a, b, c) -> verified);
             return client;
         };
-        return new FtpClient(s, getFtpConfig(port), new RetryOnExceptionStrategy());
+        RetryConfig config = new RetryConfig();
```

Comment:
2 different RetryConfig instance. one from spring one from you. better to convert this class to spring managed class.